library/src/com/google/maps/android/PolyUtil.java

```diff
@@ -172,9 +172,55 @@ public static boolean isLocationOnPath(LatLng point, List<LatLng> polyline,
 
     private static boolean isLocationOnEdgeOrPath(LatLng point, List<LatLng> poly, boolean closed,
                                                   boolean geodesic, double toleranceEarth) {
+        int idx = locationIndexOnEdgeOrPath(point, poly, closed, geodesic, toleranceEarth);
+
+        return (idx >= 0);
+    }
+
+    /**
```

Comment:
Could you please change the format here to include the `@param` and `@return` fields in the comments so the Javadocs generate correctly?  See `simplify()` for an example.  On a related note, the `poly` name in the comments doesn't match the name of any method parameters (I assume this should be `polyline`).